someSong/src/pages/home/home.html

```diff
@@ -12,40 +12,77 @@
 </ion-header>
 
 <ion-content class="no-scroll" padding="5px">
-  <ion-list-header>
-    My Questions
-  </ion-list-header>
-  <ion-scroll scrollY="true" class="myQuestions">
-    <ion-list no-lines no-padding *ngIf="userQuestions">
-      <button ion-item *ngFor="let question of userQuestions | keyValueFilter" (click)="goToQuestion(question.key)">
-        {{question.value.title}}
-        <ion-icon name="checkmark" item-left color="secondary" *ngIf="question.value.correctAnswer"></ion-icon>
-        <ion-icon name="close" item-left color="danger" *ngIf="!question.value.correctAnswer"></ion-icon>
+  <div class="scrollableContent">
+    <ion-list-header class="list-header">
+      My Questions
+    </ion-list-header>
+    <ion-scroll scrollY="true">
+      <ion-list no-lines no-padding *ngIf="userQuestions && !isEmpty(userQuestions) && !questionLoading">
+        <button ion-item *ngFor="let question of userQuestions | keyValueFilter" (click)="goToQuestion(question.key)">
+          {{question.value.title}}
+          <ion-icon name="checkmark" item-left color="secondary" *ngIf="question.value.correctAnswer"></ion-icon>
+          <ion-icon name="help" item-left color="dark" *ngIf="!question.value.correctAnswer"></ion-icon>
+        </button>
+      </ion-list>
+      <div *ngIf="userQuestions && isEmpty(userQuestions) && !questionLoading" class="contentCentered">
+        <div>
+          No questions asked yet. <br>
+          Press the
+          <ion-icon name="md-help"></ion-icon>
```

Comment:
how about using the `<ion-icon name="help-circle"></ion-icon>`?
it looks more like the "ask a question" button